barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.hpp

```diff
@@ -716,79 +714,34 @@ template <typename PCS> class ZeroMorphVerifier_ {
-                                      const std::vector<RefVector<Commitment>>& concatenation_group_commitments = {},
-                                      RefSpan<FF> concatenated_evaluations = {})
-    {
-        Commitment first_g1;
-
-        if constexpr (Curve::is_stdlib_type) {
-            auto builder = multivariate_challenge[0].get_context();
-            first_g1 = Commitment::one(builder);
-        } else {
-            first_g1 = Commitment::one();
-        }
-        auto opening_claim = compute_univariate_evaluation_opening_claim(unshifted_commitments,
-                                                                         to_be_shifted_commitments,
-                                                                         unshifted_evaluations,
-                                                                         shifted_evaluations,
-                                                                         multivariate_challenge,
-                                                                         first_g1,
-                                                                         transcript,
-                                                                         concatenation_group_commitments,
-                                                                         concatenated_evaluations);
-        return PCS::reduce_verify(opening_claim, transcript);
-    }
-
-    /**
-     * @brief Verify a set of multilinear evaluation claims for unshifted polynomials f_i and to-be-shifted
-     * polynomials g_i.
-     *
-     * @details Identical purpose as the function above but used when the verification of the PCS evaluation protocol
-     * requires the verification key prior to the last step that is accumulated.
+     * @brief Return the univariate opening claim used to verify, in a subsequent PCS, a set of multilinear evaluation
+     * claims for unshifted polynomials f_i and to-be-shifted polynomials g_i
      *
      * @param commitments Commitments to polynomials f_i and g_i (unshifted and to-be-shifted)
      * @param claimed_evaluations Claimed evaluations v_i = f_i(u) and w_i = h_i(u) = g_i_shifted(u)
      * @param multivariate_challenge Challenge point u
      * @param transcript
      * @return VerifierAccumulator Inputs to the final PCS verification check that will be accumulated
      */
-    static VerifierAccumulator verify(RefSpan<Commitment> unshifted_commitments,
+    static OpeningClaim<Curve> verify(RefSpan<Commitment> unshifted_commitments,
                                       RefSpan<Commitment> to_be_shifted_commitments,
                                       RefSpan<FF> unshifted_evaluations,
                                       RefSpan<FF> shifted_evaluations,
                                       std::span<FF> multivariate_challenge,
-                                      const std::shared_ptr<VerifierCommitmentKey<Curve>>& vk,
+                                      Commitment first_g1,
```

Comment:
nit: a Commitment is 512 bits, it could be passed with const ref

Reply:
thanks!